pkg/storage/parameterset_test.go

```diff
@@ -15,7 +15,7 @@ import (
 )
 
 func TestNewParameterSet(t *testing.T) {
-	ps := NewParameterSet("dev", "myparams",
+	ps := NewParameterSet("dev", "myparams", "",
```

Comment:
Wouldn't it make sense assert that the description is added correctly like this test does with the other inputs?

Reply:
Correct, I'll update these